src/pages/RepoPage/ConfigTab/tabs/ConfigurationManager/ConfigurationManager.tsx

```diff
@@ -26,8 +26,8 @@ function ConfigurationManager() {
   return (
     <div className="flex flex-col gap-6 lg:w-3/4">
       <CoverageConfiguration repoConfiguration={repoConfiguration} />
-      <TestAnalyticsConfiguration repoConfiguration={repoConfiguration} />
       <BundleAnalysisConfiguration repoConfiguration={repoConfiguration} />
+      <TestAnalyticsConfiguration repoConfiguration={repoConfiguration} />
```

Comment:
To confirm: we're sticking Bundle Analysis above Test Analytics? The Figma file has the order switched.

Reply:
Yes. Discussed with Nick and Kyle this morning. Good catch! I should have called that out.